google-ads-examples/src/main/java/com/google/ads/googleads/examples/campaignmanagement/GraduateCampaignExperiment.java

```diff
@@ -0,0 +1,128 @@
+      // into the code here. See the parameter class definition above for descriptions.
+      params.customerId = Long.parseLong("INSERT_CUSTOMER_ID_HERE");
+      params.experimentId = Long.parseLong("INSERT_EXPERIMENT_ID_HERE");
+    }
+
+    GoogleAdsClient googleAdsClient;
+    try {
+      googleAdsClient = GoogleAdsClient.newBuilder().fromPropertiesFile().build();
+    } catch (FileNotFoundException fnfe) {
+      System.err.printf(
+          "Failed to load GoogleAdsClient configuration from file. Exception: %s%n", fnfe);
+      return;
+    } catch (IOException ioe) {
+      System.err.printf("Failed to create GoogleAdsClient. Exception: %s%n", ioe);
+      return;
+    }
+
+    try {
+      new GraduateCampaignExperiment()
+          .runExample(googleAdsClient, params.customerId, params.experimentId);
+    } catch (GoogleAdsException gae) {
+      // GoogleAdsException is the base class for most exceptions thrown by an API request.
+      // Instances of this exception have a message and a GoogleAdsFailure that contains a
+      // collection of GoogleAdsErrors that indicate the underlying causes of the
+      // GoogleAdsException.
+      System.err.printf(
+          "Request ID %s failed due to GoogleAdsException. Underlying errors:%n",
+          gae.getRequestId());
+      int i = 0;
+      for (GoogleAdsError googleAdsError : gae.getGoogleAdsFailure().getErrorsList()) {
+        System.err.printf("  Error %d: %s%n", i++, googleAdsError);
+      }
+    }
+  }
+
+  /** Runs the example.
+   *
+   * @param googleAdsClient the client to use.
+   * @param customerId the customer ID.
+   * @param experimentId the experiment ID.
+   */
+  private void runExample(GoogleAdsClient googleAdsClient, long customerId, long experimentId) {
+    // Graduating a campaign experiment requires a new budget. Since the budget for the base
+    // campaign has explicitly_shared set to false, the budget cannot be shared with the campaign
+    // after it is made independent by graduation.
+    CampaignBudget budget =
+        CampaignBudget.newBuilder()
+            .setName(StringValue.of("Budget #" + System.currentTimeMillis()))
+            .setAmountMicros(Int64Value.of(50_000_000))
+            .setDeliveryMethod(BudgetDeliveryMethod.STANDARD)
+            .build();
+    // Creates an operation to add the new budget.
+    CampaignBudgetOperation operation =
+        CampaignBudgetOperation.newBuilder().setCreate(budget).build();
+
+    // Creates the budget.
+    String budgetResourceName;
+    try (CampaignBudgetServiceClient budgetServiceClient =
+        googleAdsClient.getLatestVersion().createCampaignBudgetServiceClient()) {
+      MutateCampaignBudgetsResponse response =
+          budgetServiceClient.mutateCampaignBudgets(
+              String.valueOf(customerId), ImmutableList.of(operation));
+      budgetResourceName = response.getResults(0).getResourceName();
```

Comment:
The Ruby example [logs the budget resource name](https://github.com/googleads/google-ads-ruby/blob/ca971b644cbe9b62cc4f39c011ac45e07ca3e05d/examples/campaign_management/graduate_campaign_experiment.rb#L44). We should update to be consistent.